Sources/Testing/Expectations/Expectation.swift

```diff
@@ -18,7 +18,6 @@ public struct Expectation: Sendable {
   ///
   /// If this expectation passed, the value of this property is `nil` because no
   /// error mismatch occurred.
-  @_spi(ForToolsIntegrationOnly)
```

Comment:
This property should probably stay SPI for now as it's a bit hacky and probably not the final design for this data.